src/wazuh_db/wdb_upgrade.c

```diff
@@ -112,6 +113,16 @@ wdb_t * wdb_upgrade_global(wdb_t *wdb) {
         }
     }
 
+    char output[OS_MAXSTR + 1] = {0};
+
+    if (version < (int)(sizeof(UPDATES)/sizeof(char*))) {
```

Comment:
Code style: if we create a variable with this content of
`(int) (sizeof(UPDATES) / sizeof(char*)` ? It'd be easier to read the code because it's used below also.

Reply:
Variable created!